libtock/proximity.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "tock.h"
+
+#define DRIVER_NUM_PROXIMITY 0x60003
+
+#ifdef _cplusplus
+extern "C" {
+#endif
+
+// function to be called when the system call is finished
+//
+// callback       - pointer to function to be called
+// callback_args  - pointer to data provided to the callback
+int proximity_set_callback (subscribe_cb callback, void* callback_args);
+
+// Read proximity async
+int proximity_read(void);
+
+// Read proximity sync
+int proximity_read_sync(unsigned*);
+
+// Return prox value on interrupt async
+int proximity_read_on_interrupt(uint8_t lower, uint8_t upper);
+
+// Return prox value on interrupt sync
```

Comment:
Can you expand this to explain what "proximity value" is? What's the valid range? Is 0 close or far? etc

Reply:
Done.  Please let me know if I need to be more specific.

Reply:
This still needs to indicate that 0 is closer and that the range is [0,255]